### ISSUE TITLE
Two field type errors

### ISSUE THREAD

**Issue by zunder1990** [davidnewhall/unifi-poller#17]
main.go:161: [ERROR] infdb.Write(bp): {"error":"field type conflict"}
main.go:161: [ERROR] infdb.Write(bp): {"error":"partial write: field type conflict: input field \"tx_power\" on measurement \"uap_radios\" is type integer, already exists as type float dropped=16"}

The 2nd error repeats on every db write. 

My unifi site in the controller has 12 APs with 8 on and 4 that are unplugged. 

unifi version=5.11.10.0

**Comment by zunder1990** [comment]
Update:
I move the down APs to another site to hide them and then  dropped/recreated the database, after all of this the poller was happy and working.

As a part two I moved one of the APs back to the site and poller starting to error. I moved the AP back out of the site and poller started to work again.

TLDR: poller does not handle an AP down.

**Comment by davidnewhall** [comment]
Is it possible this error began after you update `unifi-poller`, or the `unifi` library? This would happen if you ran `make deps` or `dep ensure -update`. I think, at one point, I changed the tx_power to an integer and it used to be a float, [here](https://github.com/davidnewhall/unifi-poller/pull/7/files#diff-5d5295078daa589fd7ccfbfffd97cc08R199). This may be the cause of the error. If you dropped the influx database it should not return. Am I understanding correctly that is what you did, and the error came back? Thanks!

**Comment by davidnewhall** [comment]
I'm actually not sure. It looks like it was always a string and it [got changed to an int](https://github.com/davidnewhall/unifi-poller/commit/bb60dae7c7c9977cf0f051b9b220ad894473176d#diff-5d5295078daa589fd7ccfbfffd97cc08). I'm not real sure when or how it was ever a float. This is an interesting error.

**Comment by davidnewhall** [comment]
I think I found the bug. There is a place in the code where I set a field value with two different types. I created a contribution that should resolve it: https://github.com/golift/unifi/pull/4 - I need to do some more testing, but this is a start.
